src/js/redux/reducers/search/searchFiltersReducer.js

```diff
@@ -110,6 +117,20 @@ const searchFiltersReducer = (state = initialState, action) => {
             });
         }
 
+        // Program Source (TAS) Filter
+        case 'UPDATE_TREASURY_ACCOUNT_COMPONENTS': {
+            return Object.assign({}, state, {
```

Comment:
FWIW we can use the spread operator here. I assume you prefer the Object.assign. Makes no difference to me just want to acknowledge this.

we could do:

```javascript
return {
    ...state,
    treasuryAccounts: ProgramSourceFilterFunctions.updateSelectedSources(state.treasuryAccounts, action.source)
}
```